.github/workflows/cypress.yml

```
@@ -20,7 +20,11 @@ jobs:
       - name: Cypress run
         uses: cypress-io/github-action@v1
         with:
+          record: true
           build: npm run build
+        env:
+          CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

```suggestion
        env:
          CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
        with:
          record: true
          build: npm run build
```
The order meant some `with` entries were being interpreted as `env` entries.

Also, I saw this when setting up the CYPRESS_RECORD_KEY: 
> Secrets are not passed to workflows that are triggered by a pull request from a fork.

presumably that means PR runs won't get recorded.